apps/src/netsim/netsim.js

```diff
@@ -3,25 +3,28 @@
  *           This file is the main entry point for the Internet Simulator.
  */
 
-var utils = require('../utils');
 import $ from 'jquery';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import {openDialog as openInstructionsDialog} from '../redux/instructionsDialog';
+
 import {getStore} from '../redux';
+import {openDialog as openInstructionsDialog} from '../redux/instructionsDialog';
+
 var _ = require('lodash');
 /** @type {Object<string, function>} */
```

Comment:
Not entirely sure what this comment is for. Could we just remove?

Reply:
Good catch!

Looks like it was originally added in [this commit](https://github.com/code-dot-org/code-dot-org/commit/bb6e2562fb63#diff-5999b620ac14e29b4000442a7a132c62dbb12c12079381f3c43cfeeff22cfda5), alongside several other type annotations. I feel pretty comfortable saying we don't need a type annotation for this import, particularly given how ubiquitous it is in our codebase; I'll remove.